peer.go

```diff
@@ -1081,64 +1085,12 @@ out:
 			}
 
 		case *lnwire.Error:
-			key := p.addr.IdentityKey
-
-			switch {
-			// In the case of an all-zero channel ID we want to
-			// forward the error to all channels with this peer.
-			case msg.ChanID == lnwire.ConnectionWideID:
-				for chanID, chanStream := range chanMsgStreams {
-					chanStream.AddMsg(nextMsg)
-
-					// Also marked this channel as failed,
-					// so we won't try to restart it on
-					// reconnect with this peer.
-					p.failedChannels[chanID] = struct{}{}
-				}
-
-			// If the channel ID for the error message corresponds
-			// to a pending channel, then the funding manager will
```

Comment:
Nice! Have been wanting to do this for a while, likely one of the oldest lingering TOODs in the codebase.